src/jit/gentree.cpp

```diff
@@ -885,6 +885,42 @@ bool GenTreeCall::IsPure(Compiler* compiler) const
            compiler->s_helperCallProperties.IsPure(compiler->eeGetHelperNum(gtCallMethHnd));
 }
 
+bool GenTreeCall::IsSideEffectFree(Compiler* compiler, bool ignoreExceptions, bool ignoreCctors) const
+{
```

Comment:
Re: #2, never mind: I see you changed that func.

---
In reply to: [133565093](https://github.com/dotnet/coreclr/pull/12892#discussion_r133565093) [](ancestors = 133565093)